nss-day-cohort-39/nutshell-zaffre-zorillas
# Added dialog box that appears when "New Event" button is clicked

### Changes 
- Created HTML for dialog box 
- Created custom event on EventList.js that dispatches when "New Event" button is clicked
- Created event listener on EventDialog.js that renders dialog box when it hears the "newEventBtnClicked" custom event

### Testing
- Click the "New Event" button and make sure Dialog box appears
- Make sure dialog box has three fieldsets: Date, Name, and Location
(Save button not yet functional)
